modules/dbfs/databaseMock.go

```diff
@@ -69,23 +72,92 @@ func (dm *DatabaseMock) CBGetFileVersion(fileID int64) (int64, error) {
 	return dm.FileVersion[fileID], nil
 }
 
-// CBGetFileChanges is a mock of the real implementation
-func (dm *DatabaseMock) CBGetFileChanges(fileID int64) ([]string, error) {
+// ScrunchFile moves a file form the starting path to the end path
+func (dm *DatabaseMock) ScrunchFile(meta FileMeta) error {
 	dm.FunctionCallCount++
-	return dm.FileChanges[fileID], nil
+
+	_, changes, err := dm.PullFile(meta)
+	if err != nil {
+		return fmt.Errorf("Scrunching - Failed to retrieve patches and file for scrunching: %v", err)
+	}
+	if len(changes) > MaxBufferLength {
+		changes, baseFile, err := dm.getForScrunching(meta, MinBufferLength)
+		if err != nil {
+			return fmt.Errorf("Scrunching - Failed to retrieve patches and file for scrunching: %v", err)
+		}
+
```

Comment:
there is a newline between this if and the next, but none of the ones below. Maybe make them consistent.